src/cryptonote_core/uptime_proof.cpp

```diff
@@ -91,6 +93,9 @@ Proof::Proof(
                 tools::make_from_guts<eth::bls_public_key>(proof.require<std::string_view>("bk"sv));
         pop_bls =
                 tools::make_from_guts<eth::bls_signature>(proof.require<std::string_view>("bp"sv));
+        if (proof.skip_until("gh")) {
```

Comment:
Only during the transition HF, or always active from the HF and onwards?

Reply:
I think this should apply only on non-mainnet.

Reply:
Oh, I missed the `if` context Doyle is referring to here, but commented similarly in the proof generation side.